src/themes/default.css

```diff
@@ -122,16 +128,17 @@
   --bl-font-title-4-semibold: var(--bl-font-weight-semibold) var(--bl-font-title-4);
   --bl-font-title-4-bold: var(--bl-font-weight-bold) var(--bl-font-title-4);
 
-  /* Font Style: Form Label */
-  --bl-font-form-label: var(--bl-font-weight-medium) var(--bl-font-size-xs) var(--bl-font-family);
+  /* Font Style: Body Text 1 */
+  --bl-font-body-text-1: var(--bl-font-weight-regular) var(--bl-font-size-s) var(--bl-font-family);
+
+  /* Font Style: Body Text 2 */
+  --bl-font-body-text-2: var(--bl-font-weight-regular) var(--bl-font-size-m) var(--bl-font-family);
+
+  /* Font Style: Body Text 3 */
+  --bl-font-body-text-3: var(--bl-font-weight-regular) var(--bl-font-size-l) var(--bl-font-family);
```

Comment:
Since there is no regular, medium, semibold variants of body texts, I feel naming them with small, medium and large makes more sense to me. But this is not a big deal for me, if you still find numbers are fine, it's ok.

Reply:
We gave numbers for `header` typographies too. We can continue this way.